Issue 2232 Added Custom CSS tab

Marko64 opened the pull request:
Fixes #2232 
- Added Custom CSS tab for all existing blocks
- Fixed getCategoriesCss() function to show custom CSS for options that exist. It was inverted.

Marko64 committed:
Added Custom CSS tab

- Added Custom CSS tab for all existing blocks
- Fixed getCategoriesCss() option to show custom CSS for options that exist, it was inverted

Naaaaiix reviewed (commented):
Very good, in terms of code looks fantastic. Also have to say @elzadj left it really easy 😜 

Apart of the comments left on code, couple of points:
1. May we should decide if we should put `:before` or `::before`; I prefer second option which corresponds to CSS 3 version.
2. I suspect @Marko64 and possibly @Rustam198804 are not working on a VSC environment with Prettier, Beautify and ESLint. We may do a document with the musts we need and the optional VSC extensions 👍

Marko64 committed:
Fixed incorrect selectors and unified pseudo-elements

- Fixed column styles.js and custom-css.js to target everything properly from the Custom CSS tab in this block.
- Tested all our blocks and did the same for a few other blocks that were not working correctly. Now they are all tested and working.
- Unified pseudo-elements, ie, changed all :before to ::before ,and :after to :;after

Marko64 committed:
Added the comment for external dependencies in SVG block

- Forgot to add this as suggested. Adding now.

elzadj reviewed (changes requested):
Text Block and some other blocks do not have Canvas, so the labels must be 'Before text', 'After text', 'Custom CSS for text ::before' etc., not '... canvas':
![Edit-Page-‹-Local-Wordpress-—-WordPress (84)](https://user-images.githubusercontent.com/2401618/144225198-27559a36-fef9-45ab-ac70-8205f5e541b0.jpg)

Marko64 committed:
Added missing green colour and removed canvas settings from the text block

Marko64 committed:
Fixed conflict

Marko64 committed:
Update editor.scss

Marko64 committed:
Merge branch 'master' of https://github.com/yeahcan/maxi-blocks into issue-2232-adding-custom-css

elzadj reviewed (approved):
Found 2 typos in Text, everything else is great, thank you @Marko64

Marko64 committed:
Fixed typos
